orangewidget/utils/itemmodels.py

```diff
@@ -180,33 +183,21 @@ def sort(self, column: int, order: Qt.SortOrder = Qt.AscendingOrder):
         data table is left unmodified. Use mapToSourceRows()/mapFromSourceRows()
         when accessing data by row indexes.
         """
-        if QT_VERSION >= 0x50000:
-            self.layoutAboutToBeChanged.emit(
-                [], QAbstractTableModel.VerticalSortHint
-            )
-        else:
-            self.layoutAboutToBeChanged.emit()
+        indices = self._sort(column, order)
+        self.__sortColumn = -1 if column < 0 else column
+        self.__sortOrder = order
+        self.setSortIndices(indices)
+
+    def setSortIndices(self, indices):
```

Comment:
From what I can tell, the only reason for exposing this is that we need to be able to sync up the sorting between two table views (like in https://github.com/biolab/orange3/pull/5509), right?

Reply:
Um, yes. :) It simplifies OWPredictions a lot. But it also allows to use this model as "reordering proxy", where the order is not necessarily imposed by sorting. Is this useful in general? I don't know, maybe. But it doesn't hurt to have this in a separate method.